main.cpp

```diff
@@ -0,0 +1,18 @@
+#include <iostream>
```

Comment:
Later you may set up Github Actions to build and run this main program every time you push and create a PR.

Reply:
Yes, I'll set it up these days.